rollup.config.js

```diff
@@ -0,0 +1,47 @@
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import uglify from 'rollup-plugin-uglify'
+import pkg from './package.json'
+
+export default [
+  // browser-friendly UMD build
+  {
+    input: 'index.js',
+    output: {
+      name: 'vtex.msk',
```

Comment:
I think the name could be only `msk`, since this package is not scoped inside @vtex org on npm.